index.html

```diff
@@ -14,31 +14,43 @@
     href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
 </head>
 
-<body class="container">
-
-  <nav class="navbar navbar-expand-md position-sticky top-0 bg-white">
-    <div class="container toggles justify-content-md-evenly">
-      <a class="navbar-brand" href="#"><i class="fa-solid fa-won-sign fa-2xl"></i></a>
-      <button class=" btn burger d-md-none" type="button"><i class="fa-solid fa-bars"></i></button>  
-      <button type="button" class="btn d-none close-btn"><i class="fa-solid fa-xmark " ></i></button>  
-      <div class="justify-content-evenly d-none d-md-flex" id="navbarNav">
-        <ul class="navbar-nav">
-          <li class="nav-item"><a class="nav-link " aria-current="page" href="#About">About</a>
-          </li>
-          <li class="nav-item"><a class="nav-link" href="#Cards">Features</a>
-          </li>
-          <li class="nav-item"><a class="nav-link" href="#feature">Sources</a>
-          </li>
-        </ul>
-      </div>
+<body>
+  <header class="position-sticky top-0">
+    <div class="nav-1 bg-dark">
+      <ul class="d-flex float-right gap-3 m-0 justify-content-end p-3 align-items-center">
+        <li class="text-white "><i class="fa-brands fa-square-facebook"></i> </li>
+        <li class="text-white "><i class="fa-brands fa-github"></i></li>
+        <li class="text-white">Logout</li>
+        <li class="text-white">Mypage</li>
+        <li class="text-white">India</li>
+      </ul>
     </div>
-  </nav>
+    <nav class="navbar navbar-expand-md  bg-white">
+      <div class="container toggles justify-content-md-evenly">
+        <a class="navbar-brand" href="#"><i class="fa-solid fa-won-sign fa-2xl"></i></a>
+        <button class=" btn burger d-md-none" type="button"><i class="fa-solid fa-bars"></i></button>
+        <button type="button" class="btn d-none close-btn"><i class="fa-solid fa-xmark "></i></button>
+        <div class="justify-content-evenly d-none d-md-flex" id="navbarNav">
+          <ul class="navbar-nav">
+            <li class="nav-item"><a class="nav-link " aria-current="page" href="#About">About</a>
+            </li>
```

Comment:
- [ ] Your project should contain two pages `home` and `about` pages. you just added the home page, Kindly add another page for `About` 👌

- [ ] Also, make sure that it's possible to navigate between the pages, there's a link from `Home` page to `About` page and the opposite too.

![image](https://user-images.githubusercontent.com/45231276/181490645-5f2742e6-b472-4431-88a2-f2093948e98a.png)

> You can find the design for the about page [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

| About page desktop version | About page mobile version  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/181491265-dee87fda-93c3-4047-8e02-92737ce4d631.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/181491320-d5cb772f-59ab-4940-a810-1ec6f6f28825.PNG" width="500" /> |